setup.py

```diff
@@ -2,12 +2,15 @@
 
 
 setup(
-    name='sp-pytest-blockage',
-    version='0.2.1',
+    name='pytest-blockage',
+    url='https://github.com/singleplatform-eng/pytest-blockage',
+    version='0.3.0+singleplatform.1',
```

Comment:
Can you send a corresponding pr to spdj, updating tis version to this one, once its push to our mirror.

Reply:
I plan on PRing anywhere we use it to move to this version. I want to get a clean version available for the library cookiecutter.